src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,49 @@
-export const Tabs = () => {};
+import { Tab } from '../../types/Tab';
+
+type Props = {
+  tabs: Tab[];
+  selectedTab: Tab;
+  onSelected: (tab: Tab) => void;
+};
+
+export const Tabs: React.FC<Props> = ({ tabs, selectedTab, onSelected }) => {
+  return (
```

Comment:
```suggestion
export const Tabs: React.FC<Props> = ({ tabs, selectedTabId, onSelected }) => {
  // Check here if selected tab is ok
  return (
```
You need to add additional check for selectedTab here (as you did in App component). Then your tests will be passed successfully.